ios/RNCCameraRoll.mm

```diff
@@ -435,8 +437,8 @@ static void RCTResolvePromise(RCTPromiseResolveBlock resolve,
       [assets addObject:@{
         @"node": @{
           @"type": assetMediaTypeLabel, // TODO: switch to mimeType?
-          @"subTypes":assetMediaSubtypesLabel,
-          @"group_name": currentCollectionName,
```

Comment:
Looks like `currentCollectionName` is not used anymore, can you clean that up?

Reply:
Sure! Done!